warehouse/templates/manage/organization/teams.html

```diff
@@ -0,0 +1,100 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+{% extends "manage_organization_base.html" %}
+
+{% set user = request.user %}
+{% set title = gettext("Organization teams") %}
+
+{% set active_tab = 'teams' %}
+
+{% block title %}{% trans organization_name=organization.name %}Manage '{{ organization_name }}' teams{% endtrans %}{% endblock %}
+
+{% block main %}
+  <h2>{% trans team_count=organization.teams|length %}Teams ({{ team_count }}){% endtrans %}</h2>
+
+  <div class="team-list">
+    {% for team in organization.teams %}
+    <div class="team-snippet">
+      <div class="split-layout split-layout--no-wrap split-layout--wrap-on-tablet">
+        <div>
+          <h3 class="team-snippet__title">{{ team.name }}
+            {% if request.user in team.users %}
+            <a href="{{ request.route_path('manage.team.roles', organization_name=organization.normalized_name, team_name=team.normalized_name) }}" class="badge package-snippet__badge">
+              Member
+            </a>
+            {% endif %}
+          </h3>
+          <p class="package-snippet__meta">
+            {% trans member_count=team.members|length %}
```

Comment:
This seems to always be calculating as zero. I think it should be `team.users` unless the name is changed on the model to `members` as mentioned in `TODO`

Reply:
Renamed to `members` as mentioned above.